tests/utils_idyntree.py

```diff
@@ -127,9 +126,7 @@ def build(
         urdf: pathlib.Path | str,
         considered_joints: list[str] = None,
         vel_repr: VelRepr = VelRepr.Inertial,
-        gravity: npt.NDArray = dataclasses.field(
-            default_factory=lambda: np.array([0, 0, -10.0])
-        ),
+        gravity: npt.NDArray = np.array([0, 0, -10.0]),
```

Comment:
I'm only wondering why this works. Recent Python versions no longer allow this kind of initialization with mutable objects. If I recall, also numpy arrays were affected.

Reply:
I guess it's because this is the argument of a method to build the class, not the attribute declaration

Reply:
Right :) I Didn't notice I was commenting on the `build` method :) I thought it was the attribute initialization, my bad.

Reply:
I guess the original code was a copy-paste leftover, woops.